docs/src/main/asciidoc/websockets-next-reference.adoc

```diff
@@ -240,6 +240,7 @@ WebSocket Next supports _blocking_ and _non-blocking_ logic, akin to Quarkus RES
 Here are the rules governing execution:
 
 * Methods annotated with `@RunOnVirtualThread`, `@Blocking` or `@Transactional` are considered blocking.
+* All Methods in a class annotated with `@RunOnVirtualThread` are considered blocking and run on a virtual thread.
```

Comment:
```suggestion
* Methods declated on a class annotated with `@RunOnVirtualThread` are considered blocking.
```

Reply:
We should also update the bullet about Kotlin `suspend` functions...

Plus change the last bullet from _"Methods annotated with `@RunOnVirtualThread` must execute on a virtual thread, each invocation spawns a new virtual thread."_ to something like _"Methods annotated with `@RunOnVirtualThread` or declared on a class annotated with `@RunOnVirtualThread` must be executed on a virtual thread, each invocation spawns a new virtual thread."_